cmd/create.go

```diff
@@ -46,23 +54,42 @@ kn func create --runtime quarkus myfunc
 
 # Create a function project that uses a CloudEvent based function signature
 kn func create --template events myfunc
-`,
-	SuggestFor: []string{"inti", "new"},
-	PreRunE:    bindEnv("runtime", "template", "repositories", "confirm"),
-	RunE:       runCreate,
-	// TODO: autocomplate or interactive prompt for runtime and template.
+	`,
+		SuggestFor: []string{"inti", "new"},
```

Comment:
```suggestion
		SuggestFor: []string{"vreate", "creaet", "craete", "new"},
```
I think this is leftover from when the command was called `init`. Some better misspelling suggestions.

Reply:
Good catch @lance , I just glossed right over that.  Definitely a holdover from the switch from init to create